src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs

```diff
@@ -588,6 +588,63 @@ public virtual IHtmlContent GenerateAntiforgery(ViewContext viewContext)
         }
 
         /// <inheritdoc />
+        public IHtmlContent GenerateGroupsAndOptions(string optionLabel, IEnumerable<SelectListItem> selectList)
```

Comment:
Please move this back down and push your branch again. Reordering makes it impossible to see what you've changed in this method. Fine to make this move after the review is complete.